files/default/sql/ddl/updates/1.0.0.sql

```diff
@@ -326,3 +326,9 @@ DEALLOCATE PREPARE stmt1;
 -- drop jobs columns from feature group and training datasets
 ALTER TABLE `hopsworks`.`training_dataset` DROP COLUMN `job_id`;
 ALTER TABLE `hopsworks`.`feature_group` DROP COLUMN `job_id`;
+
+DROP TABLE IF EXISTS `meta_data_schemaless`;
```

Comment:
@maismail just to be on the safe side, can you add the hopsworks database here?

Reply:
done